src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -70,7 +70,6 @@ export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
   }, [parentVenueId]);
 
   const tables = venue?.config?.tables ?? TABLES;
-
```

Comment:
Probably shouldn't be removed.  Empty lines a.k.a. white space some times help distinguish code better than a wall of text.

Reply:
restored c4879fa